framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -167,6 +168,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
             // check also for themed values [BUG #5906]
             if (selfPropertyValue == null) {
               // update the appearance to make sure every themed property is up to date
+              this.$$resyncNeeded = true;
```

Comment:
I don't agree with this - magic member variables that are injected into other classes to invoke special undocumented features should not be allowed.  If delayed call to `updateAppearance()` is necessary for this mixin then there's an argument that it could be necessary elsewhere - after all it is not a requirement that `MExecutable` is used in something that consumes a `Command`, it's simply a very helpful mixin.  

Ideally the variable in `qx.ui.core.Widget` should be private or protected and operated via an API.